setup.py

```diff
@@ -160,6 +157,9 @@
         "jsonschema == 2.4.0",
         "klein == 0.2.3",
         "pyrsistent == 0.9.1",
+
+        # This is used for provisioning code.
```

Comment:
No need for this comment - effect is used in multiple places (provisioning and packaging and docs uploading).